### DISCUSSION TITLE
How to write documentation not associated with any stories/components?

### DISCUSSION THREAD

**Discussion by fabis94** [storybookjs/storybook#16083] [Documentation] [answered]
In a design system you don't just write docs for all components, you also have various pages written in free-form about how to apply branding correctly, what kind of fonts should be used etc. So essentially just documentation pages not associated with any specific story/component.

I can see that this is already possible from this example Storybook (see the first 3 pages at the top), the pages even have a different icon: https://5ccbc373887ca40020446347-mfdxqmndlo.chromatic.com/?path=/story/intro--page

But I couldn't find any resources online or in the docs about how to actually do this.

**Comment by jonniebigodes** [comment, marked as answer]
@fabis94 thanks for opening this discussion we appreciate it. But did you check [this](https://storybook.js.org/docs/react/writing-docs/mdx#documentation-only-mdx) and [this section](https://storybook.js.org/tutorials/design-systems-for-developers/react/en/document/) of the tutorial (Design systems for developers)?

**Reply by fabis94** [reply]
I did see that page, but missed that part, thanks

**Reply by jonniebigodes** [reply]
No worries, glad that I was able to provide some context and possibly answer your question. Feel free to reach out to us here, or in our Discord Server for any further questions or issue you encounter when using Storybook.

Stay safe
